frontend/src/testDefinition.js

```diff
@@ -0,0 +1,9 @@
+// definition of tests
+export const TEST_DEFINITION = {
```

Comment:
What's this used for?

Reply:
This will be used when we'll use this new action. If you take a look at the PR's screenshots, you'll see that I'm using it to specify the test that I want to get the meta data. I added that in order to avoid hard coding the test names.

It is not used yet, but it'll be used as soon as we start using this new action.